### ISSUE TITLE
npm package not working?

### ISSUE THREAD

**Issue by tazilios** [jshor/datebook#62]
Hello,

Tried to install your datebook package in my Angular application using npm (https://www.npmjs.com/package/datebook) but keep on getting the same error "ERROR in src/app/app.module.ts(66,27): error TS2307: Cannot find module 'datebook'"

Did someone succed in using it?

**Comment by mwchung24** [comment]
having the same issue

**Comment by nidhisinhaTC** [comment]
having same issue...Module not found: Error: Can't resolve 'datebook'

**Comment by AndreasRohl** [comment]
same problem here. :(

 error TS2307: Cannot find module 'datebook'.

**Comment by RonaldFernandoChica** [comment]
isnt working, somebody makes it work?

**Comment by elasticdotventures** [comment]
🦃 does not work. example code has bug (missing comma; WEEKLY)

**Comment by keyuls** [comment]
Any update on this?

**Comment by pzontrop** [comment]
an old issue but just thought I would jump in for future references. This package does not have a valid module. In fact it's not really an angular package as it also doesn't expose any services etc. However after doing an npm install you can still use the source-code by importing it into your component like:
`import Calendars from 'datebook/src/utils/calendars.js';`

This will give you access to functions such as:
```
// Define your options object first as described in the readme
Calendars.downloadIcs(options); 
Calendars.getMicrosoftCalendarUrl(options);
Calendars.getGoogleCalendarUrl(options);
Calendars.getYahooCalendarUrl(options);
```

So it's far from perfect and also seems pretty dead (just look at the amount of forks and prq's) but it still beats writing and maintaining these url and file parsers yourself.

**Comment by jshor** [comment]
This is now fixed as of v4.0.0.
